### ISSUE TITLE
[cisco ftd] Update to 2.3.0 resets listen ip settings to default

### ISSUE THREAD

**Issue by rsdrakh** [elastic/integrations#3958]
Update from 2.2.1 to 2.3.0 on Elasticsearch 8.3.2
Issue could be reproduced with a different agent policy.

The set value used to be "0.0.0.0" and was reset to "localhost".

Cisco IOS also behaves like this.
Cisco ASA does not behave like this.

**Comment by LaZyDK** [comment]
This happens when the Kibana Integration template is edited. Working as intended.

**Comment by rsdrakh** [comment]
Nothing was edited, all was used out of the box (except for the initial
config of the integration), if that is intended behaviour then it is both
not user friendly and not consistent to the behaviour of other Cisco
integrations.

LaZyDK ***@***.***> schrieb am Fr., 12. Aug. 2022, 13:43:

**Comment by LaZyDK** [comment]
Yes, you updated the integration, and with this the template was edited. The template is not edited with all upgrade, this you can see here on github. 
It is consistent with all other integrations in Elastic.

**Comment by rsdrakh** [comment]
Updating an Integration should never ever overwrite custom settings that
were there before the upgrade. Because why should it? No matter if I update
one or all policies. "Works as intended" can still be very unfriendly to
our customers.
Is this behaviour documented anywhere, while it is intended?

LaZyDK ***@***.***> schrieb am Fr., 12. Aug. 2022, 14:04:

**Comment by smnschneider** [comment]
I agree with @rsdrakh. This behavior should be reconsidered. Removing user data should not be the case at all. Why should there be a difference updating all or just one integration?

**Comment by smnschndr** [comment]
Keep it up 👍
